### PR TITLE
Invites: move /accept-invite to /my-sites/invites

### LINKED ISSUE #1101: Framework: Move /accept-invite to /lib/invites
https://github.com/Automattic/wp-calypso/pull/636#issuecomment-160309895

### PR DESCRIPTION
**Pull request by lezama**
### How to test ###

Everything in the invite process should still flow.

* checkout `update/move-accept-invite-to-invites`
* invite yourself
* go to `http://calypso.dev:3000/accept-invite/<blog_id>/<invitation_id>`

Closes #1101

### CODE REVIEW

File: client/lib/invites/accept/index.jsx [hunk 1]
```diff
@@ -8,11 +8,11 @@ import classNames from 'classnames';
 /**
  * Internal Dependencies
  */
-import InviteHeader from './invite-header';
-import LoggedInAccept from './logged-in-accept';
-import LoggedOutInvite from './logged-out-invite';
+import InviteHeader from 'components/invites/invite-header';
```
Comment (mtias) [inline client/lib/invites/accept/index.jsx]: We should avoid app-components in client/components (purer UI components). I'd place invite react components in "my-sites".

**Comment by lezama** [comment]
cc @ebinnion @roccotripaldi

File: assets/stylesheets/_components.scss [hunk 2]
```diff
@@ -177,6 +172,11 @@
 @import 'my-sites/draft/style';
 @import 'my-sites/drafts/style';
 @import 'my-sites/exporter/style';
+@import 'my-sites/invites/invite-form-header/style';
+@import 'my-sites/invites/invite-header/style';
+@import 'my-sites/invites/invite-accept/logged-in/style';
```
Comment (mtias) [inline assets/stylesheets/_components.scss]: the logged-in / logged-out things should be prefixed or become sub-components.

Comment (ebinnion) [inline assets/stylesheets/_components.scss]: Aren't they sub-components?

File: client/boot/index.js [hunk 3]
```diff
@@ -42,6 +41,7 @@ var config = require( 'config' ),
 	// The following mixins require i18n content, so must be required after i18n is initialized
 	Layout,
 	LoggedOutLayout;
+import { displayInviteAccepted } from 'lib/invites/actions';
```
Comment (ebinnion) [inline client/boot/index.js]: Can we add a newline above this?

**Comment by ebinnion** [comment]
Code wise, you already addressed my comment it looks like. Functionally, this works fine.

In testing, I found a few bugs, but I don't believe those to be related to this PR and I have created separate issues for them.

:shipit:

File: assets/stylesheets/_components.scss (hunk 2 above)
Comment (mtias) [inline assets/stylesheets/_components.scss]: A subcomponent is `invite-accept/logged-in.jsx`, the folder `logged-in` is a full component.

Comment (ebinnion) [inline assets/stylesheets/_components.scss]: So, to clarify, we either need to bring the `logged-in` and `logged-out` into the `my-sites/invites` directory, or do the sub-components?

**Pull request merged**